Update amazon-s3-connector-reference.adoc

adam-lipka opened the pull request.

adam-lipka committed:
Update amazon-s3-connector-reference.adoc

adam-lipka committed:
Updated with revisited doc

Updated with revisited doc, had to copy lines 2 - 4 manually.

dhirsch1 commented on modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc:
```diff
@@ -398,134 +375,134 @@ Copies a source object to a new destination; to copy an object, the caller's acc
-| Content Type a| String |  The destination object's Content-Type HTTP header |  |
-| Target Variable a| String |  The name of a variable that stores the output of an operation. |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the targer variable. |  `#[payload]` |
+** GLACIER |  +++one of StorageClass enumerated values, defaults to StorageClass#STANDARD+++ |  +++STANDARD+++ | 
```
Is storageClass#STANDARD correct in the published doc (View File view)?

adam-lipka replied:
It should show just `STANDARD`, probably some bug in markdown generation from javadocs. However, we should not modify it in code as it's correct in Javadoc. We can modify it by hand here but we will need to do it after every release.

dhirsch1 replied:
Please modify this by hand.

adam-lipka replied:
Fixed.

dhirsch1 commented on modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc:
```diff
@@ -684,2512 +658,2530 @@ this operation creates a zero-byte object.
+| Target Variable a| String |  +++The name of a variable to store the operation's output.+++ |  | 
+| Target Value a| String |  +++An expression to evaluate against the operation's output and store the expression outcome in the target variable+++ |  `#[payload]` | 
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  A r
```
The description for Type does not appear like it's rendering correctly in the View File view. Please check this.

adam-lipka replied:
This is what I see in View File view. Could you please advise me on what is incorrect here?
![image](https://user-images.githubusercontent.com/16899901/59603578-0d477c80-910a-11e9-8efe-9959688c5f03.png)

If I click on `Reconnect` or `Reconnect Forever` links it shows me description of these with correct comments.

dhirsch1 replied:
The issue is in the Output section for the Get Bucket Versioning Configuration.
![Screen Shot 2019-06-17 at 9 18 41 AM](https://user-images.githubusercontent.com/46829037/59607629-b62db180-90e1-11e9-8911-e54e038bea87.png)

adam-lipka replied:
Thanks! It should be list of OFF, ENABLED and SUSPENDED values but I couldn't make it look like that. I updated it and now it appears syntactically correct to me but it still renders wrong. Could you please take a look?

adam-lipka committed:
Fixed storage class

dhirsch1 committed:
Update amazon-s3-connector-reference.adoc

Trying new formatting to get the GetBucketVersioningConfiguration Output section to render correctly.

dhirsch1 committed:
Update amazon-s3-connector-reference.adoc

Tried to fix rendering of Get Bucket Versioning Configuration output list.

dhirsch1 committed:
Update amazon-s3-connector-reference.adoc

dhirsch1 committed:
Update amazon-s3-connector-reference.adoc

One more fix- changing mime to MIME in the description.

dhirsch1 reviewed (approved):
All set
